### PR TITLE
Fix latest

### PR DESCRIPTION
**Pull request by saisankargochhayat**
This is using the latest version of Gitpython which doesn't need gitdb dependency. 
Issue #322

### CODE REVIEW

**Comment by saisankargochhayat** [comment]
Moved from #323

File: requirements.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@ pipenv
 toml
 delegator.py
 pyyaml
-gitpython==2.1.8
+gitpython==3.1.0
```
Comment (fridex) [inline requirements.txt]: Please match the version specifier so we run the same version in the cluster as locally.

**Comment by saisankargochhayat** [comment]
ERROR: python-gitlab 2.0.1 has requirement requests>=2.22.0, but you'll have requests 2.18.4 which is incompatible.
ERROR: kubernetes 10.0.1 has requirement urllib3>=1.24.2, but you'll have urllib3 1.22 which is incompatible.
Also these errros come up when I try to `pip install -r requirements.txt`

**Comment by saisankargochhayat** [comment]
@fridex Should I merge this?

**Pull request merged**